release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -550,6 +550,11 @@ The most common use case for the `networkConfig` configuration setting is to set
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-host-network-interfaces-in-the-install-config.yaml-file_ipi-install-installation-workflow[Configuring host network interfaces in the install-config.yaml file].
 
+[id="ocp-4-10-networking-k8s-nmstate-operator"]
+==== Kubernetes NMState Operator is GA for bare metal installations
+
+{product-title} now provides the Kubernetes NMState Operator for bare metal installations. The Kubernetes NMState Operator is still a technology preview for all other platforms. See xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html[About the Kubernetes NMState Operator] for additional details.
```

Comment:
* technology preview -> Technology Preview
* .html -> .adoc

Reply:
@bergerhoffer Can you watit with merging. I think IBM Z and IBM Power should also be listed as GA. Both platforms have fully tested NMSTATE Operator for 4.10. 
I'll bring this up in the multi-arch program call tomorrow. 
cc: @holgwolf @lsmcfadden @jaypoulz

Reply:
No problem @SNiemann15! Changes still need to be implemented and it looks like there is a merge conflict on the PR anyways. Just keep us updated!